yb-voyager/src/utils/utils.go

```diff
@@ -446,10 +446,10 @@ func GetFSUtilizationPercentage(path string) (int, error) {
 }
 
 //read the file and return slice of csv 
-func ReadFileToSlice(listFilePath string) []string {
-	file, err := os.Open(listFilePath)
+func ReadTableNameListFromFile(filePath string) ([]string, error) {
+	file, err := os.Open(filePath)
 	if err != nil {
-		ErrExit("error opening file %s: %v", listFilePath, err)
+		return []string{}, fmt.Errorf("error opening file %s: %v", filePath, err)
```

Comment:
When there is an error, it is a convention to return zero-values of all non-error return values of a function. In this case zero-value of `[]string` is `nil` and that's what you should return along with an error.
```
if err != nil {
  return nil, fmt.Errorf(...)
}
```
Fix all such occurrences.